src/Commands/PublishCommand.php

```diff
@@ -58,7 +58,7 @@ public function handle()
         $tag = $this->option('tag');
 
         if ( ! is_null($tag)) {
-            $args['--tag'] = laravel_version('5.1') ? [$tag] : $tag;
+            $args['--tag'] = version_compare(laravel_version(), '5.1.O', '>=') ? [$tag] : $tag;
```

Comment:
Please @bonsi, can you fix the version ??

It's `'5.1.0'` instead of `'5.1.O'`, this is not a zero.

My bad :stuck_out_tongue_closed_eyes: 

Thanks again